cmd/storage_upload.go

```diff
@@ -322,3 +335,35 @@ func (c *storageClient) uploadFile(bucket, file, key, acl string) error {
 
 	return err
 }
+
+func estimatePartSize(f *os.File) (int64, error) {
```

Comment:
Since we're having all the CLI commands sharing the same package namespace, I usually avoid generic names for functions. Since it's S3-specific, I'd suggest either to rename it to `estimateStoragePartSize()` or to implement it as a method on the `*storageClient` receiver.